### PR TITLE
added support for querystring in filenames

### PR DESCRIPTION
**Pull request by pmdroid**
added support for filename with querystring like: main.js?hash=12345678

### CODE REVIEW

**Comment by halt-hammerzeit** [comment]
I'm not sure if `url.parse()` is the best solution for this: it parses the whole URL object just to extract `pathname` which is irrational.
You could write a function instead, like 
```js
extract_path(from)
{
  const question_mark_index = from.indexOf('?')
  if (question_mark_index === -1)
  {
    return from
  }
  return from.slice(0, question_mark_index)
}
```

File: source/plugin/plugin.js [hunk 1]
```diff
@@ -275,4 +275,5 @@ Webpack_isomorphic_tools_plugin.urlLoaderParser          = Webpack_isomorphic_to
 Webpack_isomorphic_tools_plugin.cssLoaderParser          = Webpack_isomorphic_tools_plugin.css_loader_parser
 Webpack_isomorphic_tools_plugin.cssModulesLoaderParser   = Webpack_isomorphic_tools_plugin.css_modules_loader_parser
 Webpack_isomorphic_tools_plugin.styleLoaderFilter        = Webpack_isomorphic_tools_plugin.style_loader_filter
-Webpack_isomorphic_tools_plugin.styleLoaderPathExtractor = Webpack_isomorphic_tools_plugin.style_loader_path_extractor
\ No newline at end of file
+Webpack_isomorphic_tools_plugin.styleLoaderPathExtractor = Webpack_isomorphic_tools_plugin.style_loader_path_extractor
+Webpack_isomorphic_tools_plugin.styleLoaderPathExtractor = Webpack_isomorphic_tools_plugin.style_loader_path_extractor
```
Comment (halt-hammerzeit) [inline source/plugin/plugin.js]: @pmdroid Ok, I guess I'll just remove that duplicate line after merging this PR

Comment (halt-hammerzeit) [inline source/plugin/plugin.js]: @pmdroid Oh, seems that the new GitHub review interface didn't actually send my comments.
I guess you'll receive them now.

Comment (halt-hammerzeit) [inline source/plugin/plugin.js]: @pmdroid Deal with that duplicate line then

**Comment by pmdroid** [comment]
@halt-hammerzeit i removed the duplicated line

**Comment by halt-hammerzeit** [comment]
Ok, it's good now.
Merging and releasing.
Thx.

**Comment by halt-hammerzeit** [comment]
Try the new version

**Pull request merged**